try {
    console.log(test)
} catch (error){
    console.log("Opa, isto não devia acontecer, contate o desenvolvedor e informe a referência do erro: ")
    console.log("")
    console.log(error)
}